apps/L1G/tinder/tinder-frontend/src/app/(auth)/signup/page.tsx

```
@@ -1,11 +1,11 @@
-import TinderLogo from '@/components/TinderLogo';
+import { ForgetPassword } from '@/components/ForgetPassword';
```

ESLint

[error] 'ForgetPassword' is defined but never used.

(no-unused-vars)